Apatira_Authman/LoginAndRegistration/mysqlconnection.py

```diff
@@ -0,0 +1,39 @@
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
Your files were edited under Authman's folder.  Please move your files to your folder and then submit another pull request.